streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -176,13 +175,13 @@ public void createTopics() throws Exception {
     }
 
     @ParameterizedTest
-    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE, StreamsConfig.EXACTLY_ONCE, StreamsConfig.EXACTLY_ONCE_V2})
+    @ValueSource(strings = {StreamsConfig.AT_LEAST_ONCE, StreamsConfig.EXACTLY_ONCE_V2})
```

Comment:
We don't have to address in this PR, but why do we have `AT_LEAST_ONCE` parameters in the EOS integration test?

Reply:
As discussed on the other PRs... need to clean this up. Filed https://issues.apache.org/jira/browse/KAFKA-17558